static/docs/commands-reference/version.md

```diff
@@ -14,11 +14,16 @@ usage: dvc version [-h] [-q | -v]
 Running the command `dvc version` outputs the following information about the
 system/environment:
 
-| Type             | Detail                                                                         |
-| ---------------- | ------------------------------------------------------------------------------ |
-| `DVC version`    | Version of DVC (along with a Git commit hash in case of a development version) |
-| `Python version` | Version of the Python being used for the project in which DVC is initialized   |
-| `Platform`       | Information about the operating system of the machine                          |
+| Type              | Detail                                                                                               |
+| ----------------- | ---------------------------------------------------------------------------------------------------- |
+| `DVC version`     | Version of DVC (along with a Git commit hash in case of a development version)                       |
+| `Python version`  | Version of the Python being used for the project in which DVC is initialized                         |
+| `Platform`        | Information about the operating system of the machine                                                |
+| `Cache`           | Type of links supported between the DVC workspace and the cache directory                            |
```

Comment:
it would beneficial to add a link to the https://dvc.org/doc/user-guide/large-dataset-optimization or to the https://dvc.org/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache